run/helm/snap/README.md

```diff
@@ -0,0 +1,233 @@
+
+To configure this Snap chart properly please refer to the section [Automation of loading plugins](#automation-of-loading-plugins).
+
+### Loading plugins manually
+
+After creating Snap daemonset from the chart with command:
+
+`$ helm install stable/snap`
+
+you can list all the pods in order to verify if Snap pods are running correctly:
+
+`$ kubectl get pods --all-namespaces`
+
+The output should contain Snap pods:
+
+```
+NAMESPACE     NAME                                    READY     STATUS    RESTARTS   AGE
+default       snap-1kc54                              1/1       Running   0          1m
+default       snap-mshwn                              1/1       Running   0          1m
+default       snap-q0mkc                              1/1       Running   0          1m
+default       snap-q51xc                              1/1       Running   0          1m
+default       snap-rqk45                              1/1       Running   0          1m
+default       snap-wcklm                              1/1       Running   0          1m
+
+```
+
+This means that we have `snapteld` daemon running inside each pod. What we need now to start monitoring of the nodes are Snap plugins.
+The example below shows the use of Snap [CPU collector plugin](https://github.com/intelsdi-x/snap-plugin-collector-cpu) and [file publisher plugin](https://github.com/intelsdi-x/snap-plugin-publisher-file). 
+
+First we need to download plugin binaries:
+
+```
+$ kubectl exec -ti <snap_pod> -- curl -fsL "https://github.com/intelsdi-x/snap-plugin-collector-cpu/releases/download/6/snap-plugin-collector-cpu_linux_x86_64" -o snap-plugin-collector-cpu
+$ kubectl exec -ti <snap_pod> -- curl -fsL "https://github.com/intelsdi-x/snap-plugin-publisher-file/releases/download/2/snap-plugin-publisher-file_linux_x86_64" -o snap-plugin-publisher-file
+``` 
+
+and load them using snaptel command:
+
+```
+$ kubectl exec -ti <snap_pod> -- snaptel plugin load snap-plugin-collector-cpu
+$ kubectl exec -ti <snap_pod> -- snaptel plugin load snap-plugin-publisher-file
+```
+
+Now running the command:
+
+`
+$ kubectl exec -ti <snap_pod> -- snaptel plugin list
+`
+
+will print information about two loaded plugins. To start the actual collection of metrics you create task.
+
+```
+$ kubectl exec -ti <snap_pod> -- curl -s0 https://raw.githubusercontent.com/intelsdi-x/snap-integration-kubernetes/master/examples/tasks/cpu-file.json
+$ kubectl exec -ti <snap_pod> -- snaptel task create -t cpu-file.json
+```
+
+Command:
+`$ kubectl exec -ti <snap_pod> -- snaptel task list`
+
+will provide information whether the metrics collection was successfull or not. Use task watch to verify that tasks have been created correctly.
+`$ kubectl exec -ti <snap_pod> -- snaptel task watch <task_id>`
+
+### Automation of loading plugins
```

Comment:
configmap.yaml has section for snapteld.conf and task.json - do we need description how to customize snapteld configuration and use configmap to load tasks?